interface/billing/payment_master.inc.php

```diff
@@ -285,10 +285,11 @@ function generate_list_payment_category($tag_name, $list_id, $currvalue, $title,
                             <input class="form-control" id='post_to_date' name='post_to_date' type='text' value="<?php echo ($screen=='new_payment') ? attr(oeFormatShortDate(date('Y-m-d'))) : attr(oeFormatShortDate($PostToDate));?>"disabled>
                         </div>
                         <div class="forms col-xs-3">
-                            <label class="control-label" for="payment_method1"><?php echo xlt('Payment Method'); ?>:</label>
-                            <input type="text" class="form-control" name="payment_method1" id="payment_method1" value="<?php $frow['data_type']=1;
-                            $frow['list_id']='payment_method';
-                            generate_print_field($frow, $PaymentMethod);?>" disabled />
+                            <label class="control-label" for="payment_method"><?php echo xlt('Payment Method'); ?>:</label>
```

Comment:
nice catch, probably meant `generate_form_field` and it somehow worked for a long time :)